templates/commands/render/action.go

```diff
@@ -47,26 +47,26 @@ type walkAndModifyVisitor func([]byte) ([]byte, error)
 //
 // If the visitor returns modified file contents for a given file, that file
 // will be overwritten with the new contents.
-func walkAndModify(ctx context.Context, rfs renderFS, scratchDir string, relPaths []model.String, v walkAndModifyVisitor) error {
+//
+// rawPaths is expected to have been cleaned already from processPaths.
+func walkAndModify(ctx context.Context, sp *stepParams, rawPaths []model.String, v walkAndModifyVisitor) error {
 	logger := logging.FromContext(ctx).With("logger", "walkAndModify")
 	seen := map[string]struct{}{}
+	rfs := sp.fs
```

Comment:
Can we eliminate the `rfs` variable and inline `sp.fs` everywhere that rfs was used? Same for `scratchDir` and `sp.scratchDir`

Reply:
Removed and inlined.